src/kontrol/prove.py

```diff
@@ -902,6 +925,67 @@ def _create_initial_account_list(
     return init_account_list
 
 
+def _create_cse_accounts(
+    foundry: Foundry, storage_fields: tuple[StorageField, ...], contract_name: str, contract_storage: KApply
+) -> tuple[list[KApply], list[KApply], KApply]:
+    # TODO: call this function recursively
+    new_accounts = []
+    new_account_constraints = []
+
+    for field in storage_fields:
+        if field.data_type.startswith('contract '):
+            contract_type = field.data_type.split(' ')[1]
+            for contract_name, contract_obj in foundry.contracts.items():
+                if contract_name.split('%')[-1] == contract_type:
+                    contract_code = KEVM.bin_runtime(KApply(f'contract_{contract_obj.name_with_path}'))
```

Comment:
The `f'contract_{contract_obj.name_with_path}'` is a K macro that was generated previously in the contracts.k file. I deprecated this earlier this week. Instead, you could fetch the bin_runtime bytecode with
```
contract_code = bytesToken(bytes.fromhex(contract_obj.deployed_bytecode))
```
or
```
contract_code = token(bytes.fromhex(contract_obj.deployed_bytecode))
```
[example here](https://github.com/runtimeverification/kontrol/blob/9fcd77bdb96a661e0bc23fe3ff49f5c5a581b6d4/src/kontrol/prove.py#L559).